django_short_urls/auth.py

```diff
@@ -11,6 +11,20 @@
 from django.http import HttpResponse
 
 
+def _safe_base64_decode(encoded):
+    """Method that base64 decodes a string, while gracefully handing insufficient padding.
```

Comment:
How can we end up with "insufficient" padding? Is it invalid in regard of a RFC / is the client faulty?